packages/peregrine/lib/talons/Image/useImage.js

```diff
@@ -5,16 +5,12 @@ import { useCallback, useMemo, useState } from 'react';
  *
  * @param {function} props.onError callback for error of loading image
  * @param {function} props.onLoad callback for load of image
- * @param {Map}      props.resourceSizes image sizes used by the browser to select the image source. Supported keys are 'small', 'medium', and 'large'.
- * @param {number}   props.resourceWidth the intrinsic width of the image & the width to request for the fallback image for browsers that don't support srcset / sizes.
+ * @param {string}   props.unconstrainedSizeKey the key in props.widths for the unconstrained / default width.
```

Comment:
This is pretty weird. Is there a reason why we can't just let `"default"` be the key here?

Reply:
It is always "default", I just didn't want every file to have to know that.

I agree it is weird though.

I could go either way on it.

Reply:
The other way I thought of doing it was `import`ing the key but then I ran into circular dependencies.

Reply:
Yep, way better: https://github.com/magento/pwa-studio/pull/1956/commits/14fca612673a17a2ec3cd795505697de2a6fbb2d.

Reply:
> The other way I thought of doing it was `import`ing the key but then I ran into circular dependencies and Peregrine talons needing to `import` Venia UI files.

Right, the dependency should flow in one direction; we shouldn't have any situations where `peregrine` imports from `venia-ui`.